examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+`montage_dp` will create a montage out of images committed to `images_dp_2` and images in the master branch of `edges_dp`. 
+This is so it's easy to verify the files being processed
+by visually inspecting the montage.
+
+The most significant change from opencv in this example is the pipeline spec for `edges_dp`,
+which has the `output_branch` attribute set to `dev`.
+
+```json hl_lines="11"
+{
+    "pipeline": {
+        "name": "edges_dp"
+    },
+    "input": {
+        "pfs": {
+            "glob": "/*",
+            "repo": "images_dp_1"
+        }
+    },
+    "output_branch": "dev",
+    "transform": {
+        "cmd": [ "python3", "/edges.py" ],
+        "image": "pachyderm/opencv"
+    }
+}
+```
+
+This means that this pipeline, 
+instead of putting its output in the `master` branch,
+will put it in a branch called `dev`.
+
+Since `montage_dp` is subscribed to the master branch of `edges_dp`,
+we can simply create a `master` branch attached to any commit in `edges_dp` to trigger the pipeline.
+
+## Example run-through
+
+### Deferred Processing
+
+You should have a Pachyderm cluster set up 
+and access to it configured from your local computer
+prior to running this example.
+
+1. Run the script `setup.sh` included in this repo.
+   It will perform the following steps for you:
+   
+   ```sh
+   pachctl create repo images_dp_1
+   pachctl create repo images_dp_2
+   pachctl create pipeline -f ./edges_dp.json 
+   pachctl create pipeline -f ./montage_dp.json 
+   pachctl put file images_dp_1@master -i ./images.txt
+   pachctl put file images_dp_1@master -i ./images2.txt
+   pachctl put file images_dp_2@master -i ./images3.txt
+   pachctl create branch edges_dp@stable_1_0 --head dev^
+   ```
+   
+2. Once the demo is loaded, 
+   look at the commits in `edges_dp`.
+   They should look something like this.
+   
+   ```sh
+   $  pachctl list commit edges_dp
+   REPO         BRANCH COMMIT                           FINISHED      SIZE PROGRESS DESCRIPTION
+   edges_dp     dev    85ca689439b943978d273b311003d1bf 2 minutes ago 0B   -         
+   edges_dp     dev    dc23aa4da6b94fefbebb6751c1c9b5a3 2 minutes ago 0B   -         
+   ```
+   
+3. Remember that the `edges_dp` pipeline outputs to the `dev` branch.
+   Since the `montage_dp` pipeline subscribes to the master branch,
+   it won't be triggered when `edges_dp` jobs complete,
+   since that output goes into the `dev` branch.
+
+4. See that there are two commits in `edges_dp`,
+   both in the `dev` branch that's the output for the pipeline.
+
+   ```sh
+   $ pachctl list commit edges_dp
+   REPO         BRANCH COMMIT                           FINISHED      SIZE PROGRESS DESCRIPTION
+   edges_dp     dev    85ca689439b943978d273b311003d1bf 2 minutes ago 0B   -         
+   edges_dp     dev    dc23aa4da6b94fefbebb6751c1c9b5a3 2 minutes ago 0B   -         
+   ```
+    
+5. We've also created a stable branch that points at that first commit,
```

Comment:
IMO, this example should omit all the "stable" branch stuff. That's too specific to customer use case and isn't something that all users of deferred processing will necessarily want to do. It's also not strickly needed for this example so lets simplify things.

Reply:
yeah on reading the rest, let's just get rid of stable in this example. Move master to `dev` things get triggered, wooohoo, then keep making dev commits, etc. The stable part overcomplicates things a bit. This might require minor edits to make the flow make sense, but should be pretty easy.